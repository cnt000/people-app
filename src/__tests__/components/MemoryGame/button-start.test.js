import React from 'react'
import ReactDOM from 'react-dom'
import ButtonStart from '../../../components/MemoryGame/button-start'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

describe('ButtonStart', () => {
  it('should render (without snapshot)', () => {
    const div = document.createElement('div')
    ReactDOM.render(<ButtonStart onClick={() => console.log('click')} />, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  it('should render correctly with snapshot', () => {
    const output = shallow(<ButtonStart onClick={() => console.log('click')} />)
    expect(shallowToJson(output)).toMatchSnapshot()
  })
})
