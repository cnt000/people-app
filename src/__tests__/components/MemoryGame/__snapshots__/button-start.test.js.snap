// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonStart should render correctly with snapshot 1`] = `
<styled.div>
  <h1>
    Memory Game
  </h1>
  <button
    className="start-button"
    onClick={[Function]}
  >
    Click to start
  </button>
</styled.div>
`;
